matter/wifi/wf200/efr_spi.c

```diff
@@ -392,4 +344,4 @@ sl_status_t sl_wfx_host_disable_spi(void)
   return SL_STATUS_OK;
 }
 
-#endif
+#endif //SL_WFX_USE_SPI
```

Comment:
Since you've added the comment, could you please remove the complete #ifdef ? It doesn't make any sense for an `#ifdef` to encapsulate the whole content of a c file. If the board doesn't support SPI, the right way would be to remove the file completely from the the build system

At line 45 : `#ifdef SL_WFX_USE_SPI`

Reply:
removed #ifdef. Right now we have not added code to disable this file from the build system if the board doesn't support SPI.